#use wml::debian::translation-check translation="1.3" mindelta="1"
<define-tag description>Förfalskning av DNS-svar</define-tag>
<define-tag moreinfo>
<p>
Man har upptäckt flera sårbarheter i PyDNS, en DNS-klient implementerad i
språket Python.
Dan Kaminsky identifierade en praktisk angreppsvektor för förfalskning av
DNS-svar och förgiftning av cachen, vilka utnyttjade den begränsade entropin
i ett DNS-transaktions-id och avsaknaden av UDP-källkodsslumpning i många
implementationer av DNS.
Scott Kitterman noterade att python-dns var sårbart för denna förutsägbarhet
eftersom den varken slumpar sitt transaktions-id eller källport.
Sett tillsammans gör denna avsaknad av entropi program som använder
python-dns för DNS-förfrågningar högst mottagliga för förfalskning av svar.
</p>

<p>
Projektet Common Vulnerabilities and Exposures identifierar denna klass av
sårbarheter som
<a href="http://security-tracker.debian.org/tracker/CVE-2008-1447">CVE-2008-1447</a>
och denna specifika instans i PyDNS som
<a href="http://security-tracker.debian.org/tracker/CVE-2008-4099">CVE-2008-4099</a>.
</p>

<p>För den stabila utgåvan (Etch) har dessa problem rättats i
version 2.3.0-5.2+etch1.</p>

<p>Vi rekommenderar att ni uppgraderar ert python-dns-paket.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2008/dsa-1619.data"
